### PR TITLE
Fix issue with installing hackatime mac os

### PR DESCRIPTION
**Pull request by cshariq**
In many instances VS Code on Mac OS might not add the 'code' command to path(with existing installation and with a reinstallation), this causes the installation of hackatime to be unsuccessful. An easy fix for this is to identify if VS Code is installed, if it is and the 'code' command persists to not work, it gives a more clear error(states that the 'code' command isn't in path). I also acknowledged that many might not have the direct knowledge to add the command to path, thus to make the process more efficient once this issue has been identified it will ask Mac OS users if they'd like to add the 'code' command to path. In my experience, this issue has been common, I've tried installing hackatime with VS Code already installed on multiple devices and encountered the same issue.

### CODE REVIEW

File: public/scripts/hackatime-install.sh [hunk 1]
```diff
@@ -36,78 +50,24 @@ configure_wakatime() {
     echo
 }
 
-install_code_command() {
```
Comment (DwifteJB) [inline public/scripts/hackatime-install.sh:39]: didn't this automatically install code to bin?

Comment (cshariq) [reply public/scripts/hackatime-install.sh:39]: In my experience no, after running the script there was no new file containing the paths(was a new mac, so nth had been setup yet thus there we're no existing paths so creating the code command should have resulted in a new file) and on my other mac which alr had other paths stored, when I went to the zhsprofile file(and its equivalent files that may store paths) I couldn't locate the code command, had to add it manually. I did verify on both laptops that my script works. Maybe you want to integrate it as a backup? Also note that I didn't get any error stating that the code command couldn't be added to path.

**Comment by cytronicoder** [comment]
Hey! Can you try out [this script](https://gist.githubusercontent.com/cytronicoder/de508d8d8b45ab57fd4b3153d31735f5/raw/faa97fb3388c508e2f1ed46c3464198526b5872c/hackatime-install.sh) from #896?

**Comment by malted** [comment]
@cytronicoder @cshariq I'm kind of confused - which PR do you think should be merged?

**Comment by cshariq** [comment]
> @cytronicoder @cshariq I'm kind of confused - which PR do you think should be merged?

They're basically the same, but i think mine has simpler prompts where, as @cytronicoder has a prompt, beginners might not understand. Tbh, both will work though 😂 .

**Comment by malted** [comment]
I want to merge this but need more assurance it works - it's really confusing to read. I don't want to break installation; which cases have you tested with? OSs, installation states etc

**Comment by cshariq** [comment]
> I want to merge this but need more assurance it works - it's really confusing to read. I don't want to break installation; which cases have you tested with? OSs, installation states etc

I've tested it on 2 mac os devices with and without vs code and the code command. As far as I've tested I haven't encountered any issues.

**Comment by malted** [comment]
Thank you sm for your help with this! :)

**Pull request merged**